### ISSUE TITLE
Jiting low-rank Sinkhorn doesn't speed up computations

### ISSUE THREAD

**Issue by theouscidda6** [ott-jax/ott#73]
It seems that the low-rank alternative of Sinkhorn (https://ott-jax.readthedocs.io/en/latest/notebooks/LRSinkhorn.html) does not take advantage of `jit`. Even so, it seems that using `jit` does slow down the calculations. 

Here is a colab notebook that illustrates this aspect: https://colab.research.google.com/drive/1QAlY9x5BfJhM_DBD7WOFrf_FuNnWeAek?usp=sharing. This notebook also shows that unlike low-rank Sinkhorn, Sinkhorn is well accelerated when jited, via experiments on the Sinkhorn divergence. 

Do you have an explanation?

Thank you very much for your help.

**Comment by michalk8** [comment]
Hi @theouscidda6 ,

the sinkhorn loop is by default jitted, that's why you don't see any differences, see [here](https://github.com/ott-jax/ott/blob/main/ott/core/sinkhorn_lr.py#L215).
Using
```python
solver = ott.core.sinkhorn_lr.LRSinkhorn(
    rank=rank, max_iterations=max_iter, gamma=gamma,
    jit=False,
    use_danskin=True)
```
in your example, I get `1 loop, best of 5: 3.2 s per loop` for non-jitted version and `1 loop, best of 5: 1.41 s per loop` for the explicitly jitted one.

**Comment by theouscidda6** [comment]
Thank you very much for your answer! Everything is very clear now.
